### DISCUSSION TITLE
Storybook and Angular

### DISCUSSION THREAD

**Discussion by rfortunati** [storybookjs/storybook#28698] [Help]
### Summary

When I add storybook to a new angular project and i run this. Than I can't scroll in the documentation of the example stories also when I update the button component not the complete html output is shown in the documentation. 
<img width="1718" alt="Screenshot 2024-07-23 at 10 07 02" src="https://github.com/user-attachments/assets/883d66ea-a931-4bd3-af04-79c4382c0590">

**Comment by Marklb** [comment]
Without more information, it is hard to answer. Could you provide a reproduction? A quick simple reproduction can be made with https://storybook.new

My assumption for the scrollbar is that your apps stylesheet has page scrolling disabled. The docs pages render components inline, by default, so your stylesheets get added to it. There are a few possible solutions, depending on your project, so it is hard to say which will work without knowing what your style rules are. The most generic may be to add styles to a `.storybook/preview-head.html` that have higher precedence than your global styles and changes the styles back to default.

I can't really answer your question about your button output, because I don't know how your story is defined and I don't know what you consider *complete html* to be. The code shown, when clicking "Show Code", should be the code that someone could copy to get that story's output in their app. It is a documentation feature, so it shows what someone would want to see if they are going to use that component. Someone may have an addon for more output, but I can only see that being useful for development and devtools can show that.

**Reply by rfortunati** [reply]
@Marklb I have added a default setup in the attached file
[example-angular-ionic-storybook.zip](https://github.com/user-attachments/files/16358956/example-angular-ionic-storybook.zip)

it's the default button component story when adding storybook. I mean the preview in the documentation page is not showing the complete styled button output you only see the half of it. And I can't scroll through the documentation.
